packages/contracts-bedrock/contracts/libraries/SafeCall.sol

```diff
@@ -52,16 +65,23 @@ library SafeCall {
         bytes memory _calldata
     ) internal returns (bool) {
         bool _success;
+        bool _halt = checkMinGas(_minGas, 0);
         assembly {
-            // Assertion: gasleft() >= ((_minGas + 200) * 64) / 63
+            // Assertion: gasleft() >= (_minGas * 64) / 63 + 40_000
```

Comment:
This comment should be on `checkMinGas` above.

Some or all of the rest comment block below also needs to be moved or edited to acommodate this change.

Reply:
Still in the process of splitting these - will do.

Reply:
Comments updated, RFC